sound/soc/sof/loader.c

```diff
@@ -192,6 +192,55 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
+				   const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_config_data *config =
+		container_of(hdr, struct sof_ext_man_config_data, hdr);
+
+	/* save pointer to adsp config data to make it accessible from any place in code */
+	sdev->adsp_config = config;
+	return 0;
+}
+
+static int get_config_value(const struct sof_ext_man_config_data *config, uint32_t token,
+			    uint32_t *value)
+{
+	const struct sof_config_elem *elem;
+	const struct sof_config_elem *last;
+	int elems_counter;
```

Comment:
looks like an unused variable, you should be getting compiler warnings

Reply:
now, with `for` loop, this variable is used, removed `*elem` and `*last`